File: challenges/3-area-trapezio/c++/lucashe4rt/trapezio.cpp
#include <iostream>

int main()
{
    float baseMaior, baseMenor, altura;

    std::cout << "Insira o valor da base maior: ";
    std::cin >> baseMaior;
    std::cout << "Insira o valor da base menor: ";
    std::cin >> baseMenor;
    std::cout << "Insira o valor da altura: ";
    std::cin >> altura;

    std::cout << "A área do trapézio é: " << (baseMaior + baseMenor) * altura / 2 << std::endl;
}

File: challenges/3-area-trapezio/python/melissanied/trapezio.py
try:
  print("Programa para calular a área de um trapézio :)")
  print("Insira o valor da base maior")
  baseMaior = float(input())
  print("Insira o valor da base menor")
  baseMenor = float(input())
  print("Insira o valor da altura")
  altura = float(input())

  area = (baseMaior + baseMenor) * altura / 2

  print("A área do trapézio é", area)

except ValueError:
  print("O valor inserido não é um número, apenas números são válidos")
